exercises/static/exercises/drone_cat_mouse/web-template/hal.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
         rospy.init_node("HAL_cat")
 
         self.image = None
-        self.cat = DroneWrapper(name="rqt", ns="cat/")
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure